addons/gdUnit3/src/GdUnitTestSuite.gd

```diff
@@ -104,6 +104,10 @@ func verify(obj, times := 1):
 func verify_no_interactions(obj):
 	return GdUnitMock.verify_no_interactions(obj)
 
+# Verifies the given mock or spy has any unverified interaction.
+func verify_no_more_interactions(obj, expect_result :int = GdUnitAssert.EXPECT_SUCCESS) -> GdUnitAssert:
+	return GdUnitMock.verify_no_more_interactions(obj, expect_result)
```

Comment:
create followup to rewrite the verification functions to allow fluentable usage on mock|spy.
exceptional if expect_result  =  GdUnitAssert.EXPECT_FAIL return the assertion for failure report testing like here is made